### ISSUE TITLE
Move all services and web spa app to the Docker containers and run them from docker-compose

### ISSUE THREAD

**Issue by kakarotto67** [kakarotto67/mlmc#10]
TODO:
- Move Services and SPA into docker
- Configure docker-compose accordingly

**Comment by kakarotto67** [comment]
TODO:
- Cannot connect to containerized RabbitMQ from another container
- Cannot connect to containerized MongoDB from another container

**Comment by kakarotto67** [comment]
Experience notes:
- Used two separate docker-compose files. One to run database and bus, another to run services and webspa. This is to make sure that services and spa are run after database and bus are 100% ready.
- Reference one container from another might be tricky.
  - To reference mongo db I had to set this environment variable in compose file - `OperationDatabaseSettings__Host: host.docker.internal`
  - To reference bus I had to set this environment variable in compose file - `MessageBusSettings__HostName: message-bus` (where `message-bus` is exact name of the bus service in compose file)
  - To self-reference MGCC.API I had to use just `localhost` since it is exposed under default `80` port inside the container
